.circleci/config.yml

```diff
@@ -43,8 +43,7 @@ jobs:
           command:
```

Comment:
<img width="632" alt="image" src="https://user-images.githubusercontent.com/5394573/182258057-c5c9d04b-91bd-4514-836f-37e99755fd5b.png">
 paddings slightly off, compared to master